test/staging/Intl402/Temporal/old/time-toLocaleString.js

```diff
@@ -7,6 +7,12 @@ description: time.toLocaleString()
 features: [Temporal]
 ---*/
 
+// Tolerate implementation variance by expecting consistency without being prescriptive.
+const usDayPeriodSpace =
+  new Intl.DateTimeFormat("en-US", { timeStyle: "short" })
```

Comment:
Shouldn't this be used in the assertions, or are we looking at a difference between "en-US" and "en"?